packages/block-library/src/post-template/block.json

```diff
@@ -43,6 +41,14 @@
 			"__experimentalDefaultControls": {
 				"fontSize": true
 			}
+		},
+		"spacing": {
+			"blockGap": {
+				"__experimentalDefault": "1.5rem"
```

Comment:
Just double-checking, and it looks like the old `.is-flex-container` gap value is `1.25em`. Should we retain that for consistency, so that if someone goes to edit an existing page on a site without blockGap support, there's no change to the gap?

Reply:
yeah that's a good point!